src/components/Notifications/JoinColumnSelectionTable/JoinColumnSelectionTable.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './JoinColumnSelectionTable.scss'
+import { Select } from '../../Select'
+import { Icon } from '../../Icon'
+import { isListQuery, isAggregation, getGroupableColumns, getStringColumnIndices } from 'autoql-fe-utils'
+export default class JoinColumnSelectionTable extends React.Component {
+  constructor(props) {
+    super(props)
+    const selectableJoinColumnsLength = this.getSelectableJoinColumnsLength()
```

Comment:
can this be removed and used on line 13?

Reply:
Yep sure